cleverhans/experimental/certification/utils.py

```diff
@@ -45,6 +45,9 @@ def initialize_dual(neural_net_params_object, init_dual_file=None,
   lambda_quad = []
   lambda_lu = []
 
```

Comment:
Here I made some changes so `nu` will be a Variable, rather than a reshape of a Variable, just so the type checking will be easier to write